shell/client/grain-client.js

```diff
@@ -39,6 +39,18 @@ const promptNewTitle = function () {
   }
 };
 
+const showWhoHasAccess = function () {
+  const closer = globalTopbar.addItem({
+    name: "who-has-access",
+    template: Template.whoHasAccess,
+    popupTemplate: Template.whoHasAccessPopup,
+    startOpen: true,
+    onDismiss: function () {
+      return "remove";
+    }
```

Comment:
Transpiling removes it, and it's the recommended style in the airbnb style guide: https://github.com/airbnb/javascript#commas--dangling